app/src/main/assets/high_risk_error_style.css

```
@@ -125,66 +99,12 @@ button.inProgress {
   flex-direction: column;
 }
 
-/* Expanders have a structure of
- * <div collapsed="true/false">
- *   <h2 class="expander">Title</h2>
- *   <p>Content</p>
- * </div>
- *
- * This shows an arrow to the right of the h2 element, and hides the content when collapsed="true". */
-.expander {
-  margin: var(--moz-vertical-spacing) 0;
-  background-image: url("chrome://browser/skin/images/dropmarker.svg");
-  background-repeat: no-repeat;
-  /* dropmarker.svg is 10x7. Ensure that its centered in the middle of an 18x18 box */
-  background-position: 3px 5.5px;
-  background-size: 10px 7px;
-  padding-left: 18px;
-}
-
-div[collapsed="true"] > .expander {
-  background-image: url("chrome://browser/skin/images/dropmarker-right.svg");
-  /* dropmarker.svg is 7x10. Ensure that its centered in the middle of an 18x18 box */
-  background-size: 7px 10px;
-  background-position: 5.5px 4px;
-}
-
-div[hidden] > .expander,
-div[hidden] > .expander + *,
-div[collapsed="true"] > .expander + * {
-  display: none;
-}
-
-.blockedsite h1 {
-  border-bottom-color: #9b2e2e;
-}
-
-.blockedsite button {
-  background-color: #9b2e2e;
-  color: white;
-}
-
-/* Style warning button to look like a small text link in the
-   bottom. This is preferable to just using a text link
-   since there is already a mechanism in browser.js for trapping
-   oncommand events from unprivileged chrome pages (ErrorPageEventHandler).*/
-#ignoreWarningButton {
-  width: calc(100% + 40px);
-  -moz-appearance: none;
-  background: #b14646;
-  border: none;
-  text-decoration: underline;
-  margin: 0;
-  margin-inline-start: -20px;
-  font-size: smaller;
-  border-radius: 0;
-}
-
 /* On large screen devices (hopefully a 7+ inch tablet, we already center content (see #errorPageContainer above).
    Apply tablet specific styles here */
 @media (min-width: 550px) {
   button {
-    min-width: 160px;
+    margin: var(--moz-vertical-spacing) auto var(--moz-vertical-spacing);
```

You can shorten `margin: top horizontal bottom` to `margin: vertical horizontal` :)
```suggestion
    margin: var(--moz-vertical-spacing) auto;
```